Test_OS-Collector.json

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
Should this be called example-tree.json or something similar?

Reply:
We are buidling an SA tools box and want to have test nodes to call inner trees